doc/api/vm.md

```diff
@@ -308,31 +308,6 @@ console.log(util.inspect(sandbox));
 // { globalVar: 1024 }
 ```
 
-## vm.runInDebugContext(code)
-<!-- YAML
-added: v0.11.14
--->
-
-> Stability: 0 - Deprecated. An alternative is in development.
```

Comment:
You’ll also want to update the entry in `deprecations.md` to say `Type: End-of-Life`